boards/litex/arty/README.md

```diff
@@ -35,13 +35,13 @@ hardware multiplication and compressed instruction support (such that
 it is compatible with the `rv32imc` arch).
 
 Prebuilt and tested bitstreams (including the generated VexRiscv CPU
-Verilog files) can be obtained from the [Tock on LiteX companion
-repository
+Verilog files and a patched LiteX version to support them) can be
+obtained from the [Tock on LiteX companion repository
 releases](https://github.com/lschuermann/tock-litex/releases/). The
 current board definition has been verified to work with [release
-2021100501](https://github.com/lschuermann/tock-litex/releases/tag/2021100501). The
-bitstream for this board is located in `digilent_arty_a7-35t.zip` or
-`digilent_arty_a7-100t.zip` under `gateware/digilent_arty.bit`.
+2022081701](https://github.com/lschuermann/tock-litex/releases/tag/2022081701).
```

Comment:
Ah! should the CI tag then be a derivative of this tag? `tock-CI-from-2022081701` or s/t like that?

Reply:
Yes, that seems like a great idea! I'll rename the ref.

Reply:
Renamed the tag: `2022081701-tock-ci-0`, which is a lot more meaningfull hopefully.